page-factory-api/src/main/java/ru/sbtqa/tag/pagefactory/exceptions/NoSuchActionException.java

```diff
@@ -0,0 +1,29 @@
+package ru.sbtqa.tag.pagefactory.exceptions;
+
+public class NoSuchActionException extends AssertionError {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this class to remove "Exception" or correct its inheritance. ([squid:S2166](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS2166))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=133&issues=AWfPt2r34tUZZzFFj6Hf&open=AWfPt2r34tUZZzFFj6Hf)

[ISSUE_KEY]: AWfPt2r34tUZZzFFj6Hf